mbedtls-platform-support/src/lib.rs

```diff
@@ -106,3 +106,30 @@ pub unsafe extern "C" fn mbedtls_time(tp: *mut mbedtls_sys::types::time_t) -> mb
     }
     timestamp
 }
+
+#[cfg(target_env = "sgx")]
+#[doc(hidden)]
+#[no_mangle]
+// needs to be pub for global visibility
+pub unsafe extern "C" fn explicit_bzero(buf: *mut mbedtls_sys::types::raw_types::c_void, len: mbedtls_sys::types::size_t)  {
+    use zeroize::Zeroize;
+    let buffer = core::slice::from_raw_parts_mut(buf as *mut mbedtls_sys::types::raw_types::c_char, len);
+    buffer.zeroize();
+}
+
+/// You need to call `psa_crypto_init()` before calling any function from the SSL/TLS, X.509 or PK modules.
+/// This function is fine to be called mutiple times while ensure underlying initilization function is only
+/// been called only once.
+#[cfg(feature = "tls13")]
+pub fn psa_crypto_init() {
+    use once_cell::sync::OnceCell;
+    static INIT: OnceCell<()> = OnceCell::new();
+
+    INIT.get_or_init(|| {
+        unsafe { mbedtls_sys::psa::crypto_init() };
+        return ();
+    });
+}
+
+#[cfg(all(any(feature = "rdrand", target_env = "sgx"), feature = "tls13"))]
```

Comment:
This should only depend on whether mbedtls-sys was built with `MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG`. You can use features passed up through the build scripts.